rust/windows-client/src-tauri/src/client/gui/system_tray_menu.rs

```diff
@@ -73,12 +45,13 @@ pub(crate) fn signed_in(user_name: &str, resources: &[Resource]) -> SystemTrayMe
         .add_native_item(SystemTrayMenuItem::Separator)
         .add_item(CustomMenuItem::new("".to_string(), "Resources").disabled());
 
-    for Resource { id, name, pastable } in resources {
+    for res in resources {
+        let id = res.id();
         let submenu = SystemTrayMenu::new().add_item(CustomMenuItem::new(
             format!("/resource/{id}"),
-            pastable.to_string(),
+            res.pastable().to_string(),
```

Comment:
I think you don't need to call `to_string` here, `Cow<str>` already implements `Into<String>`